test/registry.c

```diff
@@ -56,6 +56,8 @@ struct test_layout {
     const char *variant;
     const char *brief;
     const char *description;
+    const char *iso639[3];
```

Comment:
I suggest some comment here like

> Language list (ISO-639 three-letter codes). Can be any length but we only need up to 3 in tests.

and maybe also change the name to `language_list` or `iso639s` to indicate this is a list not a single item.

(Same for `iso3166`).